lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -157,14 +171,8 @@ _InsertionLocation _insertionLocationForPackageImport(
   );
 }
 
-String decideImportFormat(bool hasPackageImports, bool hasRelativeImports) {
-  if (hasPackageImports && !hasRelativeImports) {
-    return 'package';
-  } else if (!hasPackageImports && hasRelativeImports) {
-    return 'relative';
-  } else {
-    return 'package';
-  }
+bool decideImportFormat(bool shouldUsePackageImports, bool shouldUseRelativeImports) {
```

Comment:
This probably should have a name that indicates what the boolean result means. Like shouldUsePackageImports (but then maybe the parameter by that name needs to change). And a comment to describe the decision.